test/test/fast_iterator.cpp

```diff
@@ -45,7 +45,7 @@ TEST_CASE("fast_iterator")
         REQUIRE(vbegin[2] == 4);
         REQUIRE(vbegin + 2 > vbegin);
         REQUIRE(2 + vbegin > vbegin);
-        REQUIRE(2 - (vbegin + 4) > vbegin);
+        REQUIRE(vbegin + 4 - 2 > vbegin);
```

Comment:
Did this break?

Reply:
Yes, this was testing the bogus "integer - iterator" operation. It probably makes more sense to delete the test case than to change it into a redundant test of other operations.

Reply:
Yes, it was the bogus version. It's already covered down below (`vbegin + 2 - 2`), so removing this line makes sense